File: sdk/src/main/java/org/maibot/sdk/net/WsRouter.java
package org.maibot.sdk.net;

import io.netty.channel.SimpleChannelInboundHandler;
import io.netty.handler.codec.http.FullHttpRequest;

public abstract class WsRouter extends SimpleChannelInboundHandler<FullHttpRequest> {
    abstract public void registerProcessor(WsProcessors wsProcessors);
}

File: sdk/src/main/java/org/maibot/sdk/net/WsRouter.kt
package org.maibot.sdk.net

import io.netty.channel.SimpleChannelInboundHandler
import io.netty.handler.codec.http.FullHttpRequest

abstract class WsRouter : SimpleChannelInboundHandler<FullHttpRequest>() {
    abstract fun registerProcessor(wsProcessors: WsProcessors)
}
